tests/integration/cqlengine/model/test_model_io.py

```diff
@@ -393,6 +394,27 @@ def test_previous_value_tracking_of_persisted_instance(self):
         assert self.instance.get_changed_columns() == []
         assert self.instance._values['count'].previous_value == 3
 
+        # Change Multiple values
+        self.instance.count = 4
+        self.instance.text = "changed"
+        assert len(self.instance.get_changed_columns()) == 2
+        assert 'text' in self.instance.get_changed_columns()
+        assert 'count' in self.instance.get_changed_columns()
```

Comment:
for new code, please use the TestCase asserts. I know cqlengine tests are full of these, but I've been trying to fix tests I touch, and not propagate it.